en/docs/tutorials/create-your-first-scheduled-task.md

```diff
@@ -0,0 +1,221 @@
+   
+        When the **World Bank by ballerinax** connector appears in the search results, click on it.
+   
+    3. In the **Connector** panel that appears on the right of the page, click **Continue to Invoke API**.
+   
+    4. To get the population of a specific country, select **getPopulationByCountry** in the **Operation** field.
+   
+    5. To specify the USA as the country for which the population data should be fetched, enter `"USA"` in the **CountryCode** field.
+   
+    6. Click **Save**.
+   
+5. Let's add a variable to cast the output of the World Bank connector by following the procedure below:
+
+    1. Click the last **+** icon in the low-code diagram, and then click **Variable**.
+   
+    2. In the **Variable** panel that opens on the right of the page, enter information as follows:
+
+        | **Field**            | **Value**                        |
+        |----------------------|----------------------------------|
+        | **Variable Type**    | `anydata`                        |
+        | **Variable Name**    | `payload`                        |
+        | **Value Expression** | `getPopulationByCountryResponse` |
+   
+        The expression refers to the `getPopulationByCountryResponse` variable from the World Bank connector and casts that value.
+
+    3. Click **Save**.
+
+6. Now let's add a log statement to cast the payload generated via the last variable as follows:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Click **Log** to add a log statement.
+   
+    3. In the **Expression** field, enter `payload.toString()`. This gets the value of the `payload` variable statement you added, converts it to a string format, and logs it.
+   
+    4. Click **Save**.
+
+Now you have completed designing the functionality of the scheduled task.
+
+The low-code diagram looks as follows:
+
+![Scheduled task low-code view](../assets/img/tutorials/scheduled-task/scheduled-task-low-code-view.png){.cInlineImage-half}
+
+The pro-code view looks as follows:
+
+!!! tip
+    To access the pro-code view, click the **<**/**>** icon.
+
+```
+import ballerina/log;
+import ballerinax/worldbank;
+
+public function main() returns error? {
+
+    worldbank:Client worldbankEndpoint = check new ({});
+    worldbank:IndicatorInformation[] getPopulationByCountryResponse = check worldbankEndpoint->getPopulationByCountry("USA");
+    anydata payload = getPopulationByCountryResponse;
+    log:printInfo((payload).toString());
+}
+```
+
+### Step 1.3: Run the scheduled task
+
+To check whether the scheduled task configuration you created is free of errors, click **Run**.
+
+![Run scheduled task](../assets/img/tutorials/scheduled-task/run-scheduled-task.png){.cInlineImage-half}
+
+If the scheduled task configuration is free of errors, the following appears in the terminal log.
+
+ ```
+ Running executable
+ ```
+
+In addition, a single log is printed with population statistics of the USA.
```

Comment:
In addition? I think we need to emphasise that the success criteria is receiving this log.